src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -332,3 +333,17 @@ def streams_for_huc12s(huc12s, drb=False):
         cursor.execute(sql, [tuple(huc12s)])
 
         return [row[0] for row in cursor.fetchall()]  # List of GeoJSON strings
+
+
+def drexel_fast_zonal(geojson, dataset):
+    payload = '{{"geom": {}, "rasters": ["{}"]}}'.format(geojson, dataset)
```

Comment:
This looks a little awkward, but is done to avoid converting geojson (which is a stringified JSON object) to a dict and back to a string for the request payload.